dsp-role/handler/src/main/scala/dsp/role/handler/RoleHandler.scala

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.role.handler
+
+import dsp.errors.NotFoundException
+import dsp.errors.RequestRejectedException
+import dsp.role.api.RoleRepo
+import dsp.role.domain.Role
+import dsp.role.domain.RoleUser
+import dsp.valueobjects.Id.RoleId
+import dsp.valueobjects.Permission
+import dsp.valueobjects.Role._
+import zio.ZIO
+import zio._
+
+/**
+ * The role handler.
+ *
+ * @param repo  the role repository
+ */
+final case class RoleHandler(repo: RoleRepo) {
+
+  /**
+   * Retrieves all roles (sorted by IRI).
```

Comment:
Return values/types are missing in all docstrings in this file.

Reply:
True, but I've noticed we do not always have it there (also in the new code - check user slice). We should agree on the details level and keep it the same everywhere.

Reply:
Added `@return` for every method in this file.